edit-post/components/meta-boxes/meta-boxes-area/style.scss

```diff
@@ -2,10 +2,17 @@
 .edit-post-meta-boxes-area {
 	position: relative;
 
+    /** The wordpress default for most meta-box elements is content-box. Some elements, such as textarea and input, are
```

Comment:
Spaces should be tab.

I recommend installing an [EditorConfig plugin](http://editorconfig.org/#download) for your preferred editor, which will automatically adjust to using tabs for the project specifically.

Reply:
Updated in 52dab6e